packages/design-system-website/README.md

```diff
@@ -20,4 +20,10 @@ The Puppet Design System is a cross-functional team effort with shared ownership
 
 ### Feedback
 
-We are continuing to iterate on the PDS, and appreciate your feedback and questions. Say hi in Slack in the [#team-design-system](https://puppet.slack.com/messages/CFFECRQAY) channel, contact [puppet-design-system@puppet.com](mailto:puppet-design-system@puppet.com) or [report an issue](https://tickets.puppetlabs.com/secure/CreateIssueDetails!init.jspa?pid=16902&issuetype=1&priority=6) in Jira (the “PDS” project).
+We are continuing to iterate on the PDS. The current focus is on internal use, but we always appreciate everyone's feedback and questions. Please [report issues](https://github.com/puppetlabs/design-system/issues) you find or contact [puppet-design-system@puppet.com](mailto:puppet-design-system@puppet.com).
+
+## Internal use
+
+For background on the project, see [Puppet Design System](https://confluence.puppetlabs.com/display/PDS) on Confluence or watch the [Big Picture presentation](https://primetime.bluejeans.com/a2m/events/playback/33fcd61c-3ad2-4413-9393-cc216551d61b).
```

Comment:
:thinking: will we still have access to these bluejeans recordings after the zoom switch is complete?

Reply:
From @puppet-travis:

> Hey @binford2k - those recordings are not going anywhere for the time being. My team is still working on the solution to where we’ll host those in the future once our contract with BlueJeans is over.